index.html

```diff
@@ -12,30 +12,25 @@
 <body>
     <header class="header">
         <nav class="nav">
-            <a href="/" class="john-doe">John Doe</a>
+            <a href="/" class="john-doe">Ademibowale</a>
             <nav class="header-bar"><i class=" fa fa-bars cursor "></i></nav>
         </nav>
     </header>
     <section class="main-section ">
         <div>
             <h1 class="h1 ">Hey There.</h1>
-            <h2 class="h2">I'm Adebowale</h2>
+            <h2 class="h2">I'm Ademibowale</h2>
             <h3 class="h3">I'm a Software Developer</h3>
         </div>
         <div>
             <p class="paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         </div>
         <ul class="social-icon">
-            <li><img src=" images/vector.png3.png " class="social-logo" alt="my logo "></li>
-            <li>
-                <div>
-                    <img src="images/vector.png " alt="my logo " class="social-logo logo">
-                    <img src="images/vector.png90.png " alt="my logo " class="social-logo">
-                </div>
-            </li>
-            <li><img src="images/vector.png8.png " alt="my logo " class="social-logo"></li>
-            <li><img src="images/vector.png4.png " alt="my logo " class="social-logo"></li>
-            <li><img src="images/vector.png2.png " alt="my logo " class="social-logo"></li>
+            <li><img src="images/vector.png3.png" class="social-logo" alt="my logo"></li>
+            <li><img src="images/vector.png" class="social-logo logon" alt="my logo"></li>
+            <li><img src="images/vector.png8.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png4.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png2.png" alt="my logo" class="social-logo"></li>
```

Comment:
- [ ] I can't see your social media icons as the previous reviewer asked  please try to fix this issue 

![image](https://user-images.githubusercontent.com/61790587/169910600-c4379234-f79f-4ff8-8d48-8ee2615f1be8.png)

Reply:
Implemented 👍